cmd/deploy/start.go

```diff
@@ -106,6 +120,54 @@ func start(cmd *cobra.Command, configuration *config.Configuration, options *dep
 	if *configuration.GetIsTest() {
 		utils.ConfigureLoggingForTesting(cmd)
 	}
+	deployClient := deployment.GetDeployClient(configuration)
+	var startResp *de.StartPipelineResponse
+	var rawResp *_nethttp.Response
+	var err error
+	if deployment.IsURL(options.deploymentFile) {
+		startResp, rawResp, err = WithURL(options, deployClient)
+	} else {
+		startResp, rawResp, err = FromLocalFile(cmd, options, deployClient)
+	}
+
+	if err != nil && errors.Is(err, ErrYamlFileRead) {
+		return err
+	}
+	// create response object
+	deploy := newDeployStartResponse(startResp, rawResp, err)
+	storeCommandResult(cmd, DeployResultDeploymentID, deploy.DeploymentId)
+
+	if options.waitForCompletion && err == nil {
```

Comment:
I don't think we have any end-to-end tests on this behavior - can you please confirm that this hasn't been broken?

Reply:
Affirmative - still working. Tested with -o json and plain text, with URL and without.